tv_shows_app/src/components/features/shows/ShowDetails.tsx

```diff
@@ -0,0 +1,46 @@
+import { Flex, Image, Text} from "@chakra-ui/react"
+
+interface IShowDetailsProps {
+    avgRating: number
+}
+
+export const ShowDetais = ({avgRating}: IShowDetailsProps) => {
```

Comment:
Bolje je mozda proslijediti show (u kojem azuriramo rating), na taj način nemamo ništa hardkodirano